src/ocean/core/buffer/NoIndirections.d

```diff
@@ -121,7 +121,7 @@ template NoIndirectionsBufferImpl ( )
 
     ***************************************************************************/
 
-    void opCatAssign ( in T rhs )
+    void opOpAssign (string op) ( in T rhs ) if (op == "~")
```

Comment:
Needs to update documentation

Reply:
How so ? `op` can only add one value. Also `T` isn't documented as well. Should I add the trivial documentation for both of them ?

Reply:
I think only for `op` as `rhs` is documented and it isn't needed for `T`.
In previous PRs `op` was documented as `op = operation to perform`